test/sheet-data/post-permissions.spec.ts

```
@@ -0,0 +1,65 @@
+			// This tests the case where a user can write but not read
+			const resp = await fetch(`${BASE_URL}/api/sheets/writeonly-sheet/data`, {
+				method: 'POST',
+				headers: {
+					'Content-Type': 'application/json',
+					'Authorization': 'Bearer writeonly-token'
+				},
+				body: JSON.stringify({ name: 'Test' })
+			});
+			
+			expect(resp.status).toBe(200);
+			const data = await resp.json() as { success: boolean; data: {} };
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)